### ISSUE TITLE
Problems in reproducing results

### ISSUE THREAD

**Issue by fmthoker** [LongLong-Jing/3DRotNet#2]
Thanks for the amazing work in self-supervision. I was trying to
reproduce the results and ran into some problems. I
modified the code from
https://github.com/kenshohara/3D-ResNets-PyTorch for self-supervision
task as mentioned in your paper. In particular, I am trying to
reproduce the result of Kinetics dataset with 100000 training videos.
However, my model seems to converge quickly and results in
overfitting.  I am unable to find any bug. Did you make any other
changes (like momentum, optimizer parameters, weight decay etc) to the code of 3D Resnet? Could you share some more details about the implementation?

 Here are the training and validation logs for 7 epochs. For validation, I am using ucf101 dataset.

Training logs
epoch   loss    acc     lr
1        0.801      0.60     0.1
2       0.553      0.68    0.1
3       0.487          0.70      0.1
4       0.461        0.71     0.1
5       0.443       0.71      0.1
6       0.434      0.72     0.1
7       0.430         0.72     0.1

Validaiton loss
epoch   loss    acc
1       1.44      0.282
2       1.41      0.282
3       1.78     0.246
4       1.41      0.247
5       1.49    0.243
6       1.68     0.250
7       1.45     0.256
~

**Comment by LongLong-Jing** [comment]
@fmthoker With this code, you should be able to easily reproduce the results in my paper.
